### ISSUE TITLE
Dark-green background options in the GUI

### ISSUE THREAD

**Issue by sandoche2k** [zonemaster/Zonemaster-GUI--ARCHIVED-OLD-VERSION-NOT-TO-BE-USED#153]
From Witold Kręcick

I'm wondering why the summary does not mention that the reported values
under it are on a dark green background - as I understand that means
'problematic but below warning' (eg. AXFR allowed, or no DNSSEC). If I'm
seeing a green 'check' by eg. DNSSEC for me that means everything is OK
with DNSSEC - which is misleading and confusing, as that can also mean
that there's no DNSSEC whatsoever.

Maybe there could be a fourth (besides 'OK', warning and error) level of
information saying that user should click, look and decide whether this
is a problem or not? A small icon next to the category name?

---

From Casey Decio 

 https://zonemaster.net/test/0d5cf77c3afaa6f6
The following message appears:
 "Delegation from parent to child is not properly signed (no_dnskey)"
  But it is colored green.  Is that intentional?  Or should it be red?
 Right - but this is different than a "never-signed zone".  That is, there is a distinction between zones for which there is no chain of trust (indicated by valid, authenticated non-existence proof of DS records) and where the chain is broken (indicated by unauthenticated or otherwise invalid DS non-existence proof, no matching DNSKEY, expired/invalid RRSIGs covering DNSKEY, etc.).

For example, see

https://zonemaster.net/test/0e4e25b8e9139c60
(google.com)

whose error is:

"Delegation from parent to child is not properly signed (no_ds)."

"no_ds" is much different than "no_dnskey", which is the error at:

https://zonemaster.net/test/0d5cf77c3afaa6f6
(202.in-addr.arpa)

The former indicates an insecure delegation, where the latter indicates a bogus delegation.

For comparison, note how DNSViz handles it:

http://dnsviz.net/d/google.com/Vuglyg/dnssec/
http://dnsviz.net/d/202.in-addr.arpa/VugJkg/dnssec/

Note the delegation "status" is "insecure" (black) for one and "bogus" (red) for the other.

```
The correct message is before: "No DS record had a DNSKEY with a
matching keytag."
```

Yes, this is still an error.  But sometimes the error results in a bogus delegation and sometimes it doesn't.  Which is why a status, in addition to any errors, is useful, in my opinion.  Again, you'll see in the links above that DNSViz provides both the errors as well as the status.

Just my $0.02.

**Comment by sandoche2k** [comment]
@pamasse same as color coding issue.

**Comment by pamasse** [comment]
@sandoche2k I'm not sure to understand. but i fixed the color coding issue #154

**Comment by sandoche2k** [comment]
When there is an error or warning, it should not be dark green. if the color coding issue #154 is solved, normally this should be fixed.
